tests/unit/bundles/config_example_v1/config.yaml

```diff
@@ -0,0 +1,69 @@
+- type: cluster
+  name: Cluster With Config Example
+  version: 1
+  description: |
+    This bundle is designed to provide sample of config,
+    not nessesary including all config types or combinations.
+    Don't change configs of existing objects in it,
+    add new service / component if you need.
+- type: service
+  name: with_json_fields_and_groups
+  version: 1.0
+
+  config:
+    - name: root_int
```

Comment:
what about usage of structure (list and dict)?

Reply:
more complex example, will try it with real ADCM